code/workspaces/web-app/src/actions/catalogueConfigActions.js

```diff
@@ -0,0 +1,12 @@
+import { catalogueConfig } from '../config/catalogue';
```

Comment:
Given that the contents of the json won't change without a redeploy, that means the value won't change.  Therefore, does it make sense to use the redux machinary to handle it?

Reply:
As covered in a separate discussion:
* The use of Redux came about due to a large number of components needing access to the data. Rather than each one needing to call the function that reads the configuration from file asynchronously, using Redux allows a component high-up the component tree (i.e. the admin users page) to fire the action to load the configuration asynchronously and then all the other components can read the resolved value from state without having to wait for the asynchronous resolution. This also has the benefit of being how other asynchronous state fetching is handled so it is a familiar pattern in the web-app.
* If this is the way that we think loading configuration information from config-maps should be handled, then a ticket should be created to update the other uses of config-map provided data to work with Redux.